src/libsync/propagatorjobs.cpp

```diff
@@ -181,13 +182,59 @@ void PropagateLocalMkdir::startLocalMkdir()
         done(SyncFileItem::FileNameClash, tr("Folder %1 cannot be created because of a local file or folder name clash!").arg(newDirStr), ErrorCategory::GenericError);
         return;
     }
+
+    auto parentFolderPath = std::filesystem::path{};
+    auto parentPermissions = std::filesystem::perms{};
+    try {
+        const auto newDirPath = std::filesystem::path{newDirStr.toStdWString()};
+        Q_ASSERT(newDirPath.has_parent_path());
+        parentFolderPath = newDirPath.parent_path();
+        auto parentFolderStatus = std::filesystem::status(parentFolderPath);
+        parentPermissions = parentFolderStatus.permissions();
+        if ((parentPermissions & std::filesystem::perms::owner_write) != std::filesystem::perms::owner_write) {
```

Comment:
@mgallien Maybe you could use a namespace alias for multilevel namespaces like this one?

Like namespace `stdfsperm = std::filesystem::perms;`?